### PR TITLE
Newroad Network - Task BP-01-001 -> 003 & BP-02-002 & BP-03-001 & BP-03-002 & BP-02-003 & BP-02-004 & BP-02-005 & BP-04-002

### PR DESCRIPTION
**Pull request by kvdv80**

### CODE REVIEW

File: testnet/tasks/BP-01-001/bb1d6bI3MbGC6h.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "uid": "bb1d6bI3MbGC6h",
+    "api_key_name": "NewroadNetwork"
+}
```
Comment (CloverRL) [inline testnet/tasks/BP-01-001/bb1d6bI3MbGC6h.json:4]: reviewed 01-001

Comment (CloverRL) [reply testnet/tasks/BP-01-001/bb1d6bI3MbGC6h.json:4]: reviewed 01-002

File: testnet/tasks/BP-01-003/bb1d6bI3MbGC6h.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "uid": "bb1d6bI3MbGC6h",
+}
```
Comment (CloverRL) [inline testnet/tasks/BP-01-003/bb1d6bI3MbGC6h.json:3]: reviewed 01-003

File: testnet/tasks/BP-01-001/bb1d6bI3MbGC6h.json (hunk 1 above)
Comment (0xcoco) [inline testnet/tasks/BP-01-001/bb1d6bI3MbGC6h.json:4]: reviewed 02-002

**Comment by Mummyhap** [comment]
reviewed 03-001

**Comment by Mummyhap** [comment]
reviewed 03-002

**Comment by Mummyhap** [comment]
reviewed 02004

File: testnet/tasks/BP-02-003/bb1d6bI3MbGC6h.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "provider": "0x24628c33A431173C3c71E6df91883f5162CdA582"
+}
```
Comment (Mummyhap) [inline testnet/tasks/BP-02-003/bb1d6bI3MbGC6h.json:3]: reviewed 02-003

File: testnet/tasks/BP-02-005/bb1d6bI3MbGC6h.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "provider": "0x13C178696083E44D9386Fdd35d92D3BdeC173856"
+}
```
Comment (Mummyhap) [inline testnet/tasks/BP-02-005/bb1d6bI3MbGC6h.json:3]: reviewed 02-005

**Comment by Mummyhap** [comment]
reviewed 04-002

**Pull request closed without merging**